.travis.yml

```diff
@@ -12,6 +12,7 @@ virtualenv:
   system_site_packages: true
 before_install:
   - 'for e in /.packer-env/*; do echo -n "${e}: "; cat "${e}"; done'
+  - printenv | sort # dump Travis environment for debugging
```

Comment:
@conorsch Do we want to leave this change in permanently? Seems like it was included for debugging while this work was in progress, and may no longer be needed.